homework-projects/Awais_Assignment/new-2.adoc

```diff
@@ -1,53 +1,53 @@
-[[included-file-template-2]]
-= Included file template #2
-
-*Replace me:* This is a included file template #2 with a sample cross reference to xref:template-included-file-1.adoc[Included file template #1]. +
-
-Taskwarrior
-
-[[Introduction]]
-=== Introduction: +
-Taskwarrior is a free and open-source command-line task management tool. It helps
-users to manage, organize, and prioritize their daily tasks more efficiently. It helps users
-to keep track of their tasks and critical due dates. It allows users to focus and avoid
-distractions. Here are some features of Taskwarrior:
-• Priority management
-• Scheduling
-• Time tracking
-• Customization according to individual needs
-• Integration with other tools such as calendars, email, etc.
-
-image::Graphic.jpg[300,200] 
-For more information, kindly visit https://taskwarrior.org/[github.] +
-
-
-=== Benefits: +
-Here are some of the benefits of using Taskwarrior:
-• Efficient task management.
-• Improved Productivity.
-• Taskwarrior provides a comprehensive view of all the tasks
-• It makes it easy for users to see what needs to be done first.
-• It enables users to keep track of deadlines.
-
-
-=== Users or Audiences: +
-Any individual, team, or organization can use Taskwarrior. It is significantly more
-beneficial for people working in an environment where they have to manage or deal with
-multiple tasks simultaneously.
-
-
-=== Why do users need it?: +
-Users need Taskwarrior to manage their daily workload efficiently, which reduces their
-stress levels and gives them more productivity in less time. It also helps people to
-collaborate within a team to manage group tasks and set deadlines and prioritizations. It
-also helps individuals to learn time management skills.
-How to use the product?
-To use Taskwarrior, users first need to install it on their system. Here is the official link to
-download Taskwarrior and the steps to download and use it.
-https://taskwarrior.org/download/
-
-
-=== Do users accomplish their goals?: +
-Yes, it helps users achieve their goals such as time management, reducing daily work
-stress, meeting deadlines, effective team communication, task prioritization, maintaining
-focus, and avoiding distractions.
+[[new-2]]
+= new #2
+
+Last Assignment. This is a included new #2 with a sample cross reference to xref:new-1.adoc[new #1]. +
+
+Taskwarrior
+
+
+* Introduction: +
+Taskwarrior is a free and open-source command-line task management tool. It helps
+users to manage, organize, and prioritize their daily tasks more efficiently. It helps users
+to keep track of their tasks and critical due dates. It allows users to focus and avoid
+distractions. Here are some features of Taskwarrior:
+• Priority management
+• Scheduling
+• Time tracking
+• Customization according to individual needs
+• Integration with other tools such as calendars, email, etc.
+
+image::Graphic.jpg[300,200] 
```

Comment:
You also have to commit the `Graphic.jpg` file if you want to include it in the doc as a picture.